router_bench.ts

```diff
@@ -0,0 +1,53 @@
+import { createRouter } from "./router.ts";
+import {
+  createRouteMap as createRenoMap,
+  createRouter as createReno,
+  forMethod,
+} from "https://deno.land/x/reno@v2.0.53/reno/mod.ts";
+
+const router = createRouter({
+  "/endpoint": () => new Response("Hello"),
+  "/endpoint2/:id": {
+    POST: (_req, params) =>
+      Promise.resolve(
+        new Response(JSON.stringify(params), {
+          headers: { "Content-Type": "application/json" },
+        }),
+      ),
+  },
+});
+
+const reno = createReno(createRenoMap([
```

Comment:
Codacy has a fix for the issue: [Insert `⏎··`](https://app.codacy.com/gh/httpland/http-router/pullRequest?prid=9963312)
```suggestion
const reno = createReno(
```